src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -4027,40 +4027,40 @@ components:
     ##############################################################################
     SearchIndexRequest:
```

Comment:
I renamed the items in the OpenApi spec based on @nmalfroy 's comments on the skeleton implementation.